src/iOS/Xamarin.iOS/Samples/SceneView/AnimateImageOverlay/AnimateImageOverlay.cs

```diff
@@ -0,0 +1,265 @@
+                Spacing = 10,
+                LayoutMargins = new UIEdgeInsets(5, 5, 5, 5),
+                LayoutMarginsRelativeArrangement = true,
+            };
+            stackView.AddArrangedSubview(opacityLabel);
+            stackView.AddArrangedSubview(_opacitySlider);
+
+            // Create the buttons.
+            _pauseButton = new UIBarButtonItem { Title = "Stop" };
+            _speedButton = new UIBarButtonItem { Title = "Slow" };
+
+            // Add the buttons to a toolbar.
+            _buttonToolbar.Items = new UIBarButtonItem[] {
+                new UIBarButtonItem(UIBarButtonSystemItem.FlexibleSpace),
+                _pauseButton,
+                new UIBarButtonItem(UIBarButtonSystemItem.FlexibleSpace),
+                _speedButton,
+                new UIBarButtonItem(UIBarButtonSystemItem.FlexibleSpace),
+            };
+
+            // Add the views.
+            View.AddSubviews(_mySceneView, stackView, _buttonToolbar);
+
+            // Lay out the views.
+            NSLayoutConstraint.ActivateConstraints(new[]{
+                _mySceneView.TopAnchor.ConstraintEqualTo(View.SafeAreaLayoutGuide.TopAnchor),
+                _mySceneView.BottomAnchor.ConstraintEqualTo(stackView.TopAnchor),
+                _mySceneView.LeadingAnchor.ConstraintEqualTo(View.LeadingAnchor),
+                _mySceneView.TrailingAnchor.ConstraintEqualTo(View.TrailingAnchor),
+
+                stackView.TopAnchor.ConstraintEqualTo(_mySceneView.BottomAnchor),
+                stackView.BottomAnchor.ConstraintEqualTo(_buttonToolbar.TopAnchor),
+                stackView.LeadingAnchor.ConstraintEqualTo(View.LeadingAnchor),
+                stackView.TrailingAnchor.ConstraintEqualTo(View.TrailingAnchor),
+
+                _buttonToolbar.TopAnchor.ConstraintEqualTo(stackView.BottomAnchor),
+                _buttonToolbar.BottomAnchor.ConstraintEqualTo(View.SafeAreaLayoutGuide.BottomAnchor),
+                _buttonToolbar.LeadingAnchor.ConstraintEqualTo(View.LeadingAnchor),
+                _buttonToolbar.TrailingAnchor.ConstraintEqualTo(View.TrailingAnchor),
+            });
+        }
+
+        public override void ViewWillAppear(bool animated)
+        {
+            base.ViewWillAppear(animated);
+
+            // Subscribe to events.
+            _pauseButton.Clicked += StopStartAnimation;
+            _opacitySlider.TouchUpInside += ChangeOpacity;
+            _speedButton.Clicked += SpeedButtonClick;
+        }
+
+        public override void ViewDidDisappear(bool animated)
+        {
+            base.ViewDidDisappear(animated);
+
+            // Unsubscribe from events, per best practice.
+            _pauseButton.Clicked -= StopStartAnimation;
+            _opacitySlider.TouchUpInside -= ChangeOpacity;
+            _speedButton.Clicked -= SpeedButtonClick;
+
+            // Stop the animation when the sample is unloaded.
+            _animationStopped = true;
+            _timer.Dispose();
```

Comment:
If the timer is going to be disposed here, it will need to be re-created in `ViewWillAppear`. If you try to go to the info page and return to the sample, it stops working.